modules/build/src/main/scala/scala/build/options/ClassPathOptions.scala

```diff
@@ -1,16 +1,14 @@
 package scala.build.options
 
-import dependency._
-
-import scala.build.Positioned
+import scala.build.options.collections.BuildOptionsConverterImplicits._
 
 final case class ClassPathOptions(
   extraRepositories: Seq[String] = Nil,
   extraClassPath: Seq[os.Path] = Nil,
   extraCompileOnlyJars: Seq[os.Path] = Nil,
   extraSourceJars: Seq[os.Path] = Nil,
   fetchSources: Option[Boolean] = None,
-  extraDependencies: Seq[Positioned[AnyDependency]] = Nil,
+  extraDependencies: DependencyMap = Map.empty,
```

Comment:
It seems we're losing the order of `extraDependencies`, which actually matters (it changes the order of JARs in the class path, which matters if two JARs define the same class or have resources at the same paths).